src/components/Main/Main.tsx

```diff
@@ -215,8 +215,7 @@ function Main() {
                       autorunSimulation={autorunSimulation}
                       toggleAutorun={togglePersistAutorun}
                       severity={severity}
-                      params={allParams}
-                      mitigation={allParams.containment}
+                      mitigation={scenarioState.data.containment}
```

Comment:
The mitigation interval editor modifies the scenarioState, so this is a more explicit source of truth than `allParams`. Opinion...